libraries/ssh_crypto.rb

```diff
@@ -48,7 +48,7 @@ def valid_ciphers # rubocop:disable Metrics/CyclomaticComplexity, Metrics/Method
       when /7\./
         ciphers = ciphers66
       end
-    when 'fedora'
+    when 'amazon', 'fedora'
```

Comment:
@woneill thanks for your PR! Sure its `fedora`and not `redhat`?

Reply:
It is in the `redhat` family but since it uses a newer kernel and packages than `redhat 6` I figured it'd be more accurate to have it use the `fedora` case matchers.

Reply:
@woneill does the check for `inspec.os[:release]`not cover the newer kernel/packages? I always thought that current amazon linux is based on RHEL 7, is it not the case?

Reply:
It's a weird mix of RHEL 6 and 7 features. For instance, the package versions appear to be similar to RHEL 7 releases but it doesn't use systemd. Also `inspec.os[:release]` returns a date-like value such as `2017.09`.

```
== Operating System Details

Name:      amazon
Family:    redhat
Release:   2017.09
Arch:      x86_64
```

Reply:
@woneill sounds reasonable, thanks for explanations!